docs/source/Howtos/Beginner-Tutorial/Part1/Beginner-Tutorial-Building-Quickstart.md

```diff
@@ -29,137 +28,136 @@ Will give you a list of all commands available to you. Use
+You will see how long it is until it "fires" next. Don't be alarmed if nothing happens when the countdown reaches zero &mdash; this particular script has a randomizer to determine if it will say something or not. So you will not see output every time it fires.
+
+When you are tired of your character's "insights," stop the script with:
 
     script/stop self = tutorials.bodyfunctions.BodyFunctions
 
-You create your own scripts in Python, outside the game; the path you give to `script` is literally the Python path to your script file. The [Scripts](../../../Components/Scripts.md) page explains more details.
+You may create your own scripts in Python, outside the game; the path you give to `script` is literally the Python path to your script file. The [Scripts](../../../Components/Scripts.md) page explains more details.
 
 ## Pushing Your Buttons
 
-If we get back to the box we made, there is only so much fun you can have with it at this point. It's just a dumb generic object. If you renamed it to `stone` and changed its description, no one would be the wiser. However, with the combined use of custom [Typeclasses](../../../Components/Typeclasses.md), [Scripts](../../../Components/Scripts.md)
-and object-based [Commands](../../../Components/Commands.md), you could expand it and other items to be as unique, complex
-and interactive as you want.
+If we get back to the box we made, there is only so much fun you can have with it at this point. It's just a dumb generic object. If you renamed it to `stone` and changed its description, no one would be the wiser. However, with the combined use of custom [Typeclasses](../../../Components/Typeclasses.md), [Scripts](../../../Components/Scripts.md) and object-based [Commands](../../../Components/Commands.md), you can expand it &mdash; and other items &mdash; to be as unique, complex, and interactive as you want.
 
-Let's take an example. So far we have only created objects that use the default object typeclass named simply `Object`. Let's create an object that is a little more interesting. Under
-`evennia/contrib/tutorials` there is a module `red_button.py`.  It contains the enigmatic `RedButton` class.
+So, let's work though just such an example. So far, we have only created objects that use the default object typeclass named simply `Object`. Let's create an object that is a little more interesting. Under
+`evennia/contrib/tutorials` there is a module `red_button.py`. It contains the enigmatic `RedButton` class.
 
 Let's make us one of _those_!
 
     create/drop button:tutorials.red_button.RedButton
 
-The same way we did with the Script earlier, we specify a "Python-path" to the Python code we want Evennia to use for creating the object. There you go - one red button.
+Enter the above command, switch, and Python-path and there you go &mdash; one red button! Just as in the Script example earlier, we have specified a "Python-path" to the Python code that we want Evennia to use for creating the object. 
```

Comment:
The switch is part of the command, it should not be mentioned as a separate entity here.

Reply:
indeed